lti_consumer/lti_1p1/consumer.py

```diff
@@ -138,6 +138,7 @@ def __init__(self, lti_launch_url, oauth_key, oauth_secret):
         self.lti_launch_url = lti_launch_url
         self.oauth_key = oauth_key
         self.oauth_secret = oauth_secret
+        self.xblock = xblock
```

Comment:
@pkulkark @gabor-boros I've been asked by @nedbat to take a look at this PR. 

The approach here doesn't fit the new code structure we're trying to achieve here. 
The `lti_1p1` folder should be a generic LTI 1.1 library, with no knowledge or usage of `xblock` module.